<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About App</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="PLVAnalysis.qrc">:/TT/Icons/logo.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelCaptionVersion">
         <property name="text">
          <string>Version :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCaptionBuild">
         <property name="text">
          <string>Build :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCaptionReleaseDate">
         <property name="text">
          <string>Released :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCaptionOS">
         <property name="text">
          <string>OS :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCaptionAuthor">
         <property name="text">
          <string>Author :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelVersion">
         <property name="text">
          <string>XX.XX.XX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBuild">
         <property name="text">
          <string>XX-bit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelReleaseDate">
         <property name="text">
          <string>xxxx-xx-xx</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOS">
         <property name="text">
          <string>os</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAuthor">
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="PLVAnalysis.qrc"/>
 </resources>
 <connections/>
</ui>
